backend/src/main/java/org/eclipse/tractusx/semantics/registry/service/ShellService.java

```diff
@@ -87,9 +88,142 @@ public void mapShellCollection(Shell shell){
          shell.getSubmodels().forEach( submodel -> submodel.setShellId( shell ) );
          shell.getDescriptions().forEach( description -> description.setShellId( shell ) );
          shell.getDisplayNames().forEach( description -> description.setShellId( shell ) );
+         shell.setShellExtensions(shell.getShellExtensions());
```

Comment:
Do we really need this?

Reply:
Yes. Because here we are mapping parent to child. Further is the child to parent mapping.